README.md

```diff
@@ -56,6 +56,14 @@ make styleguide-app
 
 If you'd like to test your widget in the monolith before publishing, check out the helpful tips [here](https://paper.dropbox.com/doc/Grafting-noredink-ui-into-the-monolith--AOae7awXY8Br7YEhbZunUEZdAg-ff7N9zwMyRR2D5Umza5Ez).
 
+### Public Styleguide
+
+We publish a public styleguide. To see a preview locally, run:
```

Comment:
There are references to `make styleguide-app` a few lines above this in the README. I think we should remove them, and make it clear that this is what to use when developing.